src/parser/shaman/elemental/modules/talents/LiquidMagmaTotem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+
+import { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import Statistic from 'interface/statistics/Statistic';
+
+class LiquidMagmaTotem extends Analyzer {
+  damageGained: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.LIQUID_MAGMA_TOTEM_TALENT.id);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.LIQUID_MAGMA_TOTEM_DAMAGE), this.onLMTDamage);
+  }
+
+  onLMTDamage(event: DamageEvent) {
+    this.damageGained += event.amount;
+  }
```

Comment:
remember to add ``+ (event.absorbed || 0)`` here, so we get the accurate added damage